app/javascript/packs/blockButton.js

```diff
@@ -0,0 +1,91 @@
+function initBlock() {
+  const blockButton = document.getElementById(
+    'user-profile-dropdownmenu-block-button',
+  );
+  const { profileUserId } = blockButton.dataset;
+
+  function unblock() {
+    fetch(`/user_blocks/${profileUserId}`, {
+      method: 'DELETE',
+      headers: {
+        Accept: 'application/json',
+        'X-CSRF-Token': window.csrfToken,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        user_block: {
+          blocked_id: profileUserId,
+        },
+      }),
+    })
+      .then(response => response.json())
+      .then(response => {
+        if (response.outcome === 'unblocked') {
+          blockButton.removeEventListener('click', this);
+          blockButton.innerText = 'Block';
+          blockButton.addEventListener('click', block);
+        }
+      })
+      .catch(e => {
+        window.alert(`Something went wrong: ${e}`);
+      });
+  }
+
+  function block() {
+    const confirmBlock = window.confirm(
+      'Are you sure you want to block this person? This will:\n- prevent them from commenting on your posts\n- block all notifications from them\n- prevent them from messaging you via DEV Connect',
```

Comment:
I think you can use backticks here:

```javascript
`Are you sure you want to block this person? This will:
- prevent them from commenting on your posts
- block all notifications from them
- prevent them from messaging you via DEV Connect`
```